components/board/genericlinux/digital_interrupts.go

```diff
@@ -65,19 +66,19 @@ func (b *SysfsBoard) createDigitalInterrupt(
 
 	cancelCtx, cancelFunc := context.WithCancel(ctx)
 	result := digitalInterrupt{
-		parentBoard: b,
-		interrupt:   interrupt,
-		line:        line,
-		cancelCtx:   cancelCtx,
-		cancelFunc:  cancelFunc,
-		config:      &config,
+		activeBackgroundWorkers: &b.activeBackgroundWorkers,
```

Comment:
could we perhaps rename this to something like `boardWorkerGroup`? Don't love the name I just gave in particular, but would be nice to have something that indicates the purpose of the wait group and it's source

Reply:
Yes, I'll change to `boardWorkers`. I do like the distinction that this is the board's struct, rather than our own internal version.